index.php

```diff
@@ -0,0 +1,97 @@
+<?php
+$host = 'db';
```

Comment:
All this database connection code should be in a file, which you then `require_once` into this one.